pages/add/add.jsx

```diff
@@ -20,7 +20,8 @@ export default React.createClass({
       formValues: {},
       authError: false,
       serverError: false,
-      submitting: false
+      submitting: false,
+      showFormInvalidNotice: false
```

Comment:
I renamed this from `formInvalid` to `showFormInvalidNotice` since form is still invalid at some point... it's just we decide to not show the message to user.